packages/react/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -72,7 +72,7 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>>
   public onQueryStringChanged = (queryString: string): void => {
     if (queryString !== this.state.queryString) {
       this.setState({
-        queryString: queryString,
+        queryString,
```

Comment:
This was a linting change; I was probably the first to touch this file in a while.